internal/customtypes/export_services.go

```diff
@@ -158,3 +161,18 @@ func (es *ExportServices) Merge(es2 ExportServices) error {
 
 	return es.Set(strings.Join(mergedServices, ","))
 }
+
+func ExportServicesValidValues() []string {
```

Comment:
Could we create a utility method like this that directly returns a map, if this is consistently being transformed into a map by the caller?